metadetect/metadetect.py

```diff
@@ -231,6 +255,19 @@ def _add_positions_and_psf(self, cat, res, shear_str):
             else:
                 newres['ormask'] = self.ormask[rclip, cclip]
 
+            if np.any(self.mfrac > 0):
+                newres["mfrac"] = measure_mfrac(
+                    mfrac=self.mfrac,
+                    x=newres["sx_col_noshear"],
```

Comment:
In the PR  description you said you were measuring at the "unsheared position", is this what you meant?

Reply:
Yes.